layouts/index.html

```diff
@@ -0,0 +1,57 @@
+{{ define "header" }}
+<div class="jumbotron-custom">
+  <div class="logo-ctn">
+    <img
+      class="banner-logo"
+      src="./images/bannerlogo.png"
+      alt="Entrepreneurial Open Source"
+    />
+    <a
+      class="guest-btn"
+      target="_blank"
+      href="https://docs.google.com/forms/d/e/1FAIpQLSeM9Wh9KsvppT0IFvq5nONIFFXZOqgbNP_Pl__X-saDq4qYcQ/viewform?usp=sf_link"
+    >
+      <img
+        class="dialog-box"
+        src="./images/dialogBox.svg"
+        alt="Be a guest button"
+      />
+      <span>Be A Guest</span>
+    </a>
+  </div>
+  <div class="header-ctn">
+    <h1>Entrepreneurs at the</h1>
```

Comment:
Putting 3 h1's next to each other is not the solution here. When going through a screen reader or any automated reader, it would be rendered as 3 sections that are sentence fragments. Instead, we should have a single H1 with br tags and span classes for color